src/toasts/toasts.css

```diff
@@ -12,22 +12,34 @@
   z-index: 1;
   left: 0;
   right: 0;
-  bottom: 30px;
-  font-size: 17px;
+  bottom: 2rem;
+  font-size: 1.5rem;
   word-break: break-all;
   box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.25);
 }
 
+#divider {
+  position: absolute;
+  left: 85%;
+  top: 0%;
+  bottom: 0%;
+  border-left: 1px solid white;
+}
+
+#omrs-toast-btn {
+  display: block;
+  height: 100%;
+
+  float: right;
+}
+
 #omrs-toast #text {
   margin: 16px 12px 16px 12px;
```

Comment:
@Andu033 `rem` is better to use than `px`

Reply:
Ok. I will change